tfe/resource_tfe_workspace_test.go

```diff
@@ -937,6 +938,60 @@ func TestAccTFEWorkspace_createWithRemoteStateConsumers(t *testing.T) {
 	})
 }
 
+func TestAccTFEWorkspace_createWithSourceURL(t *testing.T) {
```

Comment:
At first glance I was going to suggest refactoring the tests into table tests under one `TestAccTFEWorkspace` function, but it looks like you're just keeping consistency with how the rest of the tests are defined in this file so I decided against mentioning it.

Reply:
Yeah, I've seen all sorts of different ways of doing this in Providers. But this is how it's done here, so kept up with that standard 👍